core/shared/src/main/scala/coursier/util/JsonReport.scala

```diff
@@ -0,0 +1,50 @@
+package coursier.util
+
+import java.io.File
+
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.fasterxml.jackson.module.scala.DefaultScalaModule
+import com.fasterxml.jackson.module.scala.experimental.ScalaObjectMapper
+import coursier.Artifact
+import coursier.core.Dependency
+
+import scala.collection.mutable.ArrayBuffer
+
+
+case class JsonPrintRequirement(fileByArtifact: collection.mutable.Map[String, File], depToArtifacts: Map[Dependency, Seq[Artifact]], conflictResolutionForRoots: Map[String, String])
```

Comment:
If that class could only be added in the `cli` module, that'd be awesome! Even if that requires some code duplication in a first time…

Reply:
ok yeah. I can try that.

Reply:
moved to separate module as commented above.